### PR TITLE
Fixed suggestions, sorry about the earlier issues

### PR DESCRIPTION
**Pull request by Thomashighbaugh**

### CODE REVIEW

File: apps/opitx-md-editor/opitx.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Opitx Markdown Editor
```
Comment (HashimotoYT) [inline apps/opitx-md-editor/opitx.yml]: Rename the file to `opitx-md-editor.yml`

Comment (Thomashighbaugh) [reply apps/opitx-md-editor/opitx.yml]: fixed

**Comment by HashimotoYT** [comment]
ping @Thomashighbaugh ☝ (review request above)

**Comment by HashimotoYT** [comment]
Thanks for your interesting of adding your application into the repository, unfortunately, you don't provide required information/changes in a required time, and we need to close this PR. If you still want in adding your application please open new PR with the required information/changes.

**Pull request closed without merging**